src/ctl/ctl.c

```diff
@@ -78,17 +81,52 @@ char *Strdup(const char *s) {
     return p;
 }
 
+umf_result_t umfCtlGet(const char *name, void *ctx, void *arg) {
+    assert(name != NULL);
+    assert(arg != NULL);
+    assert(ctx != NULL);
```

Comment:
Cannot be in public API

Reply:
Done. I applied @bratpiorka idea, about returning value `UMF_RESULT_ERROR_INVALID_ARGUMENT`.